.github/workflows/contrib-openai.yml

```diff
@@ -24,6 +24,21 @@ jobs:
         python-version: ["3.10"]
     runs-on: ${{ matrix.os }}
     environment: openai1
+    services:
+      pgvector:
+        image: ankane/pgvector
+        env:
+          POSTGRES_DB: postgres
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
```

Comment:
should be added to the secrets.

Reply:
Who can help add these to secrets?

Reply:
I've added POSTGRES_PASSWORD to the secrets. You can use `${{ secrets.POSTGRES_PASSWORD }}`